Kickstarter-iOS/Views/Controllers/ChangeEmailViewController.swift

```diff
@@ -86,13 +87,22 @@ internal final class ChangeEmailViewController: UIViewController, MessageBannerV
       |> \.textColor .~ .ksr_red_400
       |> \.text %~ { _ in Strings.We_ve_been_unable_to_send_email() }
 
-    _ = self.currentEmailLabel
+    _ = self.currentEmailContainer
+      |> \.isAccessibilityElement .~ true
+      |> \.accessibilityLabel %~ { _ in
```

Comment:
I suppose the `guard` isn't necessary if we're returning `nil`? Could just be:
```swift
|> \.accessibilityLabel %~ { _ in self.currentEmailTitle.text }
```

Reply:
Oh and you could try `.~ self.currentEmailTitle.text` because this doesn't need to go through a localization function I think.

Reply:
What was I thinking 😄

Reply:
Turns out the `accessibilityLabel` has to be localized...the way we're combining these two are `accessibilityLabel` & `accessibilityValue` which reads them as one element.

We set the value on line 135 where we bind it to the VM